src/konnektor/network_planners/concatenators/mst_concatenator.py

```diff
@@ -39,7 +39,7 @@ def __init__(
             scoring function evaluating an atom mapping, and giving a score
             between [0,1].
         n_connecting_edges: int, optional
-            number of connecting edges. (default: 2)
+            maximum number of connecting edges. (default: 2)
```

Comment:
Are there cases where not exactly n_connecting_edges are being generated?

Reply:
Yes, Kruskal's algorithm will not create a cycle, so it iteratively adds edges up until either its criteria is met, or it reaches the user-provided number of connecting edges. I think we should rename this to `max_edges` or similar, but that's out of the scope of this PR.

Reply:
We can rename pre-1.0, so please raise an issue for this!

Reply:
can be addressed in [this issue](https://github.com/OpenFreeEnergy/konnektor/issues/99).